docker-compose.ssl.local.yml

```diff
@@ -36,6 +36,24 @@ services:
     volumes:
       - redis_data:/data
 
+  elasticsearch:
+    image: elasticsearch:7.17.5
+    restart: always
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    environment:
+      - node.name=es-node01
```

Comment:
We should probably consolidate all non-changing env variables into a single place (perhaps inside the folder for the new service).